tests/metadata_config_test.cc

```diff
@@ -481,6 +482,40 @@ TEST_F(MetadataConfigTest, EastWestL7LbMetadataNoOriginalSource) {
               (cilium_mark_socket_option->mark_ >> 16) == 8);
 }
 
+TEST_F(MetadataConfigTest, ProxyLibNotConfigured) {
+  ::cilium::BpfMetadata config{};
+
+  EXPECT_NO_THROW(initialize(config));
+
+  auto socket_metadata = config_->extractSocketMetadata(socket_);
+  EXPECT_TRUE(socket_metadata);
+
+  socket_metadata->configureProxyLibApplicationProtocol(socket_);
+
+  EXPECT_CALL(socket_, setRequestedApplicationProtocols(_)).Times(0);
```

Comment:
Not sure if absolutely needed, but it would be cleaner if `EXPECT_CALL` would be before the call above, like in the other test!

Reply:
changed - thanks!